cmd/kube-scheduler/app/options/options.go

```diff
@@ -252,6 +252,25 @@ func (o *Options) Config() (*schedulerappconfig.Config, error) {
 
 	c.Client = client
 	c.InformerFactory = informers.NewSharedInformerFactory(client, 0)
+
+	// if the resource provider kubeconfig is not set, default to the local cluster
+	//
+	if c.ComponentConfig.ResourceProviderClientConnection.Kubeconfig == "" || !kubeconfigFileExists(c.ComponentConfig.ResourceProviderClientConnection.Kubeconfig) {
+		klog.V(4).Infof("ResourceProvider kubeConfig is not set. default to local cluster client")
```

Comment:
we probably can set it to V(2) so this will be in the perf runs as default. it is only logged at the starting time of the scheduler process so it should be fine with V(2)

Reply:
fixed with V(2)